common/src/path.rs

```diff
@@ -179,14 +224,38 @@ pub fn clean_path(path: &str) -> Cow<'_, str> {
     Cow::Owned(rv)
 }
 
-/// Splits off the last component of a binary path.
+/// Splits off the last component of a path given as bytes.
 ///
-/// The path should be a path to a file, and not a directory. If this path is a directory or the
-/// root path, the result is undefined.
+/// The path should be a path to a file, and not a directory with a trailing directory separator. If
+/// this path is a directory or the root path, the result is undefined.
 ///
 /// This attempts to detect Windows or Unix paths and split off the last component of the path
 /// accordingly. Note that for paths with mixed slash and backslash separators this might not lead
 /// to the desired results.
+///
+/// **Note**: This is the same as [`split_path`], except that it operates on byte slices.
+///
+/// # Examples
+///
+/// Split the last component of a UNIX path:
+///
+/// ```
+/// assert_eq!(
+///     symbolic_common::split_path_bytes("/a/b/c".as_bytes()),
```

Comment:
```suggestion
///     symbolic_common::split_path_bytes(b"/a/b/c"),
```
this might be easier to read, for the other examples as well

Reply:
Updated it where it's possible. Unfortunately `b""` is not a slice, it's an array.